kalite/distributed/static/js/distributed/software-keyboard.js

```diff
@@ -96,7 +96,7 @@ window.SoftwareKeyboardView = Backbone.View.extend({
                 .appendTo(self.software_keyboard);
 
             jQuery.each(row, function(j, key) {
-                var keySpan = $("<div class='.col-xs-4'><button class='key green_button " + (key === "bs" ? "key-bs" : "") + "' value='" + key + "'>" + (key === "bs" ? "Del" : key) + "</button></div>").appendTo(rowDiv);
+                var keySpan = $("<div class='.col-xs-4'><button class='key simple-button " + (key === "bs" ? "key-bs" : "") + "' value='" + key + "'>" + (key === "bs" ? "Del" : key) + "</button></div>").appendTo(rowDiv);
```

Comment:
One thing here - the 'Del' key is slightly misaligned on the keypad. I think it is inheriting some styling that changes the font size and makes it misaligned, can you play around with it (first, just make its styling the same as all the other keys), and see if that fixes the alignment issue too?